src/libraries/System.IO.Compression/src/System/IO/Compression/ZipArchive.cs

```diff
@@ -30,92 +35,60 @@ public class ZipArchive : IDisposable
+        /// <exception cref="System.ArgumentException">The stream is already closed, or the capabilities of the stream do not match the mode.</exception>
+        /// <exception cref="System.ArgumentNullException"><paramref name="stream" /> is <see langword="null" />.</exception>
+        /// <exception cref="System.ArgumentOutOfRangeException"><paramref name="mode" /> is an invalid value.</exception>
+        /// <exception cref="System.IO.InvalidDataException">The contents of the stream could not be interpreted as a zip archive.
+        /// -or-
+        /// <paramref name="mode" /> is <see cref="System.IO.Compression.ZipArchiveMode.Update" /> and an entry is missing from the archive or is corrupt and cannot be read.
+        /// -or-
+        /// <paramref name="mode" /> is <see cref="System.IO.Compression.ZipArchiveMode.Update" /> and an entry is too large to fit into memory.</exception>
         public ZipArchive(Stream stream, ZipArchiveMode mode, bool leaveOpen) : this(stream, mode, leaveOpen, entryNameEncoding: null) { }
 
-        /// <summary>
-        /// Initializes a new instance of ZipArchive on the given stream in the specified mode, specifying whether to leave the stream open.
-        /// </summary>
-        /// <exception cref="ArgumentException">The stream is already closed. -or- mode is incompatible with the capabilities of the stream.</exception>
-        /// <exception cref="ArgumentNullException">The stream is null.</exception>
-        /// <exception cref="ArgumentOutOfRangeException">mode specified an invalid value.</exception>
-        /// <exception cref="InvalidDataException">The contents of the stream could not be interpreted as a Zip file. -or- mode is Update and an entry is missing from the archive or is corrupt and cannot be read. -or- mode is Update and an entry is too large to fit into memory.</exception>
+        /// <summary>Initializes a new instance of the <see cref="System.IO.Compression.ZipArchive" /> class on the specified stream for the specified mode, uses the specified encoding for entry names, and optionally leaves the stream open.</summary>
         /// <param name="stream">The input or output stream.</param>
-        /// <param name="mode">See the description of the ZipArchiveMode enum. Read requires the stream to support reading, Create requires the stream to support writing, and Update requires the stream to support reading, writing, and seeking.</param>
-        /// <param name="leaveOpen">true to leave the stream open upon disposing the ZipArchive, otherwise false.</param>
-        /// <param name="entryNameEncoding">The encoding to use when reading or writing entry names in this ZipArchive.
```

Comment:
Is the info in this param still relevant? Seems that it is not captured in the documentation page for this ctor

https://docs.microsoft.com/en-us/dotnet/api/system.io.compression.ziparchive.-ctor?view=net-5.0#System_IO_Compression_ZipArchive__ctor_System_IO_Stream_System_IO_Compression_ZipArchiveMode_System_Boolean_System_Text_Encoding_